doc/doxygen/src/advanced-build-system-tricks.md

```diff
@@ -38,3 +38,37 @@ You can configure your own files that will be parsed by the build system main
 * Specify a hard written `PORT` / `DEBUG_ADAPTER_ID` for some BOARD values
 * Define your custom targets
 * Override default targets
+
+
+Analyze dependency resolution                   {#analyze-depedency-resolution}
+=============================
+
+When refactoring dependency handling or modifying variables used for dependency
+resolution, one may want to evaluate the impact on the existing applications.
+This describe some debug targets to dump variables used during dependency
+resolution.
+
+To analyze one board and application run the following commands in an
+application directory.
+
+Generate the variables dump with the normal dependency resolution to a
+`dependencies_info_board_name` file:
+
+~~~~~~~~~~~~~~~~~~~
+BOARD=board_name make dependency-debug
+~~~~~~~~~~~~~~~~~~~
+
+Or with the version used by murdock to know supported boards to a
+`dependencies_info-boards-supported_board_name` file:
```

Comment:
Or with the "quick" version used by murdock (this is an incomplete resolution, details in `makefiles/dependencies_debug.inc.mk`):

Reply:
I changed it this way to still keep the rest of sentence.
I squash directly.

``` diff
diff --git a/doc/doxygen/src/advanced-build-system-tricks.md b/doc/doxygen/src/advanced-build-system-tricks.md
index b5f0e9b67..c7614a8be 100644
--- a/doc/doxygen/src/advanced-build-system-tricks.md
+++ b/doc/doxygen/src/advanced-build-system-tricks.md
@@ -58,8 +58,9 @@ Generate the variables dump with the normal dependency resolution to a
 BOARD=board_name make dependency-debug
 ~~~~~~~~~~~~~~~~~~~

-Or with the version used by murdock to know supported boards to a
-`dependencies_info-boards-supported_board_name` file:
+Or with the "quick" version used by murdock to know supported boards
+(this is an incomplete resolution, details in `makefiles/dependencies_debug.inc.mk`)
+to a `dependencies_info-boards-supported_board_name` file:

 ~~~~~~~~~~~~~~~~~~~
 BOARDS=board_name DEPENDENCY_DEBUG=1 make info-boards-supported
```

Reply:
Yep looks good!